### PR TITLE
Log unified UI usage, log deprecated classes and functions usage

### PR DESCRIPTION
**Pull request by andrews-moc**
**What was solved?**
[Developers want to send logs that will let them know how many clients use Unified UI](https://glia.atlassian.net/browse/MOB-2808)
[Developers want to send logs from deprecated functions to let them know how many clients use them](https://glia.atlassian.net/browse/MOB-2809)

**Release notes:**

 - [ ] Feature
 - [x] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

 - [ ] Tests updated? Added? Unit, acceptance, snapshots?
 - [x] Logging for future troubleshooting of client issues added?

### CODE REVIEW

**Review by AndriiHorishniiMOC — APPROVED** [review]
LGTM!

**Review by DavDo — APPROVED** [review]
LGTM

File: widgetssdk/src/main/java/com/glia/widgets/helper/Logger.kt [hunk 1]
```diff
@@ -118,6 +120,32 @@ internal object Logger {
...
+    @JvmStatic
+    fun logDeprecatedMethodUse(tag: String, methodName: String) {
+        val logMessage = DEPRECATED_METHOD_CALL_LOG_MESSAGE + methodName
+
+        if (loggingAdapters.isEmpty() && BuildConfig.DEBUG) {
+            Log.d(tag, logMessage)
+        }
+
+        loggingAdapters.forEach(Consumer { loggingAdapter: LoggingAdapter ->
+            loggingAdapter.log(LogLevel.DEBUG, tag, logMessage, concatGlobalMeta(null))
```
Comment (gugalo) [inline widgetssdk/src/main/java/com/glia/widgets/helper/Logger.kt:132]: Out of curiosity why debug logs did not work before?

**Review by gugalo — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/helper/Logger.kt (hunk 1 above)
Comment (andrews-moc) [reply widgetssdk/src/main/java/com/glia/widgets/helper/Logger.kt:132]: @gugalo I will replace `debug` log level with `info`.
The same as I did for Core SDK.
`debug` is not showing up in Kibana.

Comment (Pelkar) [reply widgetssdk/src/main/java/com/glia/widgets/helper/Logger.kt:132]: Debug is not supposed to show up in kibana, we were sending all but debug level logs to kibana

**Review by Pelkar — APPROVED** [review]
LGTM

File: widgetssdk/src/main/java/com/glia/widgets/helper/Logger.kt (hunk 1 above)
Comment (andrews-moc) [reply widgetssdk/src/main/java/com/glia/widgets/helper/Logger.kt]: @gugalo FYI, changed level to `info` now.

**Pull request merged**